### PR TITLE
Update first changelog

### PR DESCRIPTION
**Pull request by willtai**
# Description

This PR updates the CHANGELOG manually for the 0.2.0 version. The content of this existing CHANGELOG.md will then be used as a prompt into Codium which will automatically make suggestions when people open PRs in future ([see Codium code](https://github.com/Codium-ai/pr-agent/blob/main/pr_agent/tools/pr_update_changelog.py#L168)).

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [x] Documentation update
- [x] Project configuration change

## Complexity

Complexity: Low

## How Has This Been Tested?
- [ ] Unit tests
- [ ] E2E tests
- [ ] Manual tests

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [ ] Unit tests have been updated
- [ ] E2E tests have been updated
- [ ] Examples have been updated
- [ ] New files have copyright header
- [x] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# @neo4j/neo4j-genai-python
+
+## 0.2.0
```
Comment (oskarhane) [inline CHANGELOG.md:3]: We don't know the version until the actual release, could you make this "Next" or something instead?

**Review by oskarhane — APPROVED** [review]
Let's try this 🍶 
Next we need to add a script to the release action to replace "Next" the the actual version, and create a new "Next" section.

Which I can do in #54

**Comment by willtai** [comment]
@oskarhane It worked! See https://github.com/neo4j/neo4j-genai-python/pull/48#issuecomment-2149761814

**Comment by oskarhane** [comment]
Very nice!

**Comment by oskarhane** [comment]
Added the CHANGELOG update on releases in https://github.com/neo4j/neo4j-genai-python/pull/57

**Pull request merged**